provisioning/plugins/corim-nitro-decoder/test-harness/AWSNitroComid.json

```diff
@@ -0,0 +1,40 @@
+{
+    "lang": "en-GB",
+    "tag-identity": {
+      "id": "fb6789c3-147d-4e5f-8be0-4452b93c3767",
+      "version": 0
+    },
+    "entities": [
+      {
+        "name": "AWS",
+        "regid": "https://aws.amazon.com",
+        "roles": [
+          "tagCreator",
+          "creator",
+          "maintainer"
+        ]
+      }
+    ],
+    "triples": {
+      "attester-verification-keys": [
+        {
+          "environment": {
+            "class": {
+              "id": {
+                "type": "uuid",
+                "value": "51d1a77a-9908-49e0-92df-e9c872a9b0bd"
+              },
+              "vendor": "AWS",
+              "model": "Nitro"
+            }
+          },
+          "verification-keys": [
+            {
+              "key": "MIICETCCAZagAwIBAgIRAPkxdWgbkK/hHUbMtOTn+FYwCgYIKoZIzj0EAwMwSTELMAkGA1UEBhMCVVMxDzANBgNVBAoMBkFtYXpvbjEMMAoGA1UECwwDQVdTMRswGQYDVQQDDBJhd3Mubml0cm8tZW5jbGF2ZXMwHhcNMTkxMDI4MTMyODA1WhcNNDkxMDI4MTQyODA1WjBJMQswCQYDVQQGEwJVUzEPMA0GA1UECgwGQW1hem9uMQwwCgYDVQQLDANBV1MxGzAZBgNVBAMMEmF3cy5uaXRyby1lbmNsYXZlczB2MBAGByqGSM49AgEGBSuBBAAiA2IABPwCVOumCMHzaHDimtqQvkY4MpJzbolL//Zy2YlES1BR5TSksfbb48C8WBoyt7F2Bw7eEtaaP+ohG2bnUs990d0JX28TcPQXCEPZ3BABIeTPYwEoCWZEh8l5YoQwTcU/9KNCMEAwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUkCW1DdkFR+eWw5b6cp3PmanfS5YwDgYDVR0PAQH/BAQDAgGGMAoGCCqGSM49BAMDA2kAMGYCMQCjfy+Rocm9Xue4YnwWmNJVA44fA0P5W2OpYow9OYCVRaEevL8uO1XYru5xtMPWrfMCMQCi85sWBbJwKKXdS6BptQFuZbT73o/gBh1qUxl/nNr12UO8Yfwr6wPLb+6NIwLz3/Y="
```

Comment:
as I said above, this does not need to be an external endorsement: it's effectively an immutable root that is common to any possible nitro-based enclave, as such it belongs to the verification package

Reply:
> I disagree. While it's long-lived, it's possible that AWS may update it or even revoke it. That sort of thing does not belong in the package.

there is no way to revoke a root CA cert (it's the classic chicken & egg).  You can only remove it from the "root store".  and one can quite naturally consider the Go package the "root store" for the nitro attestation PKI.

Reply:
> Additionally, considering your suggestion that we use an alternate CA cert for testing (and generate "fake" AWS Nitro attestation documents for this testing), embedding it in the package seems like it would complicate things.

that sounds like an implementation detail, e.g. one could do: 

```golang
// verify.go

func verifyWithRoot(evidence []bytes, rootCA x509.Certificate) {
    // ...
}

func Verify(evidence []bytes) {
    return verifyWithRoot(evidence, AWSRootCA)
}
```
and
```golang
// verify_test.go

func TestVerify_withAltRoot_fail(t *testing.T) {
    // ...
    expectedErr := `cannot anchor path onto a trusted root`

    assert.EqualError(t, verifyWithRoot(testEvidence, altRootCA), expectedErr)
}
```

Reply:
Yes, it's not revokable, but that doesn't mean it will not be revoked. It would not be too difficult for them to send a notice to all AWS users that have used NItro enclaves. Certainly messy, but possible.

Additionally, AWS may in the future want to use different root CA certs for different situations (consider their US government cloud, which I imagine already does, if it supports Nitro at all).
I still feel that the CA Cert does not belong in the authentication library, it should be provisioned into the Veraison infra.

If the CA cert was in the authentication library, Veracruz has zero use for Veraison in the nitro use-case, because we are basically only relying on Veraison to handle provisioning and maintenance of that CA Cert.